src/Microsoft.Fx.Portability.MetadataReader/DependencyFinderEngineHelper.cs

```diff
@@ -131,8 +137,11 @@ private MemberDependency GetMemberReferenceMemberDependency(MemberReference memb
                 definedInAssemblyIdentity = _reader.FormatAssemblyInfo(memberRefInfo.ParentType.DefinedInAssembly.Value);
             }
             // If no assembly is set, then the type is either a primitive type or it's in the current assembly.
```

Comment:
This was the existing comment: if  `definedInAssemblyIdentity` member is null that means the parent is either a primitive type or defined in the current assembly. (it would be both a primitive and defined in the current assembly only in a test case, where current assembly mscorlib). Is this not accurate?